prime-react/src/main/scala/lucuma/react/primereact/Toast.scala

```diff
@@ -56,6 +58,7 @@ object Toast {
   trait Facade extends js.Object {
     def show(message:    js.Array[MessageItem]): Unit = js.native
     def replace(message: js.Array[MessageItem]): Unit = js.native
+    def remove(message:  js.Array[MessageItem]): Unit = js.native
```

Comment:
So, `remove` removes a toast based on `message`?

Reply:
It's specified like that in the API. I would have expected an `id`. Maybe it just looks at the `id`?

https://primereact.org/toast/#api.Toast.methods